content/800-data-platform/050-data-proxy/100-use-data-proxy.mdx

```diff
@@ -0,0 +1,211 @@
+
+        <img
+            src="../images/data-proxy/create-and-name-new-connection-string.png"
+            alt="Prisma Data Platform - Data Proxy - Create and name a new Data Proxy connection string"
+            width="450px"
+        />
+
+    4. Copy the `prisma://` connection string.
+
+        <Admonition type="warning">
+        
+        **Important**<br /><br />
+
+        Save the `prisma://` connection string securely. After you navigate to another page, you cannot retrieve the connection string again from the **Data Proxy**. You can only generate a new connection string, if necessary.
+
+        </Admonition>
+
+        <img
+            src="../images/data-proxy/copy-connection-string.png"
+            alt="Prisma Data Platform - Data Proxy - Copy the Data Proxy connection string"
+            width="650px"
+        />
+
+## Generate Prisma Client for the Data Proxy
+
+After you create your project in the platform and enable the Data Proxy, you can generate Prisma Client for the Data Proxy and use it with the `prisma://` connection string you obtained.
+
+You can generate and use Prisma Client for the Data Proxy in the following environments:
+
+- In your local development environment, to test the process.
+- In your deployment platform. You must replace the direct database connection string with the Data Proxy connection string and make sure that Prisma Client is auto-generated with the `--data-proxy` flag as described below.
+
+### Prerequisites
+
+* [Enable Data Proxy for your project](#enable-the-data-proxy-for-a-project).
+* Copy the Data Proxy connection string. Do so for the project environment for which you want to generate Prisma Client.
+
+### Steps
+
+1. In your local development environment, replace the direct database connection string with the Data Proxy `prisma://` connection string.
+    ```bash file=.env highlight=1;delete|2;add
+    - DATABASE_URL="postgresql://****:****@****"
+    + DATABASE_URL="prisma://****:****@****"
+    ```
+    <Admonition>
+    
+    **Note**<br /><br />
+    For security reasons, we strongly recommend that you always use environment variables for your database connection strings and Data Proxy strings. Do not hard-code them into your `schema.prisma` file.<br /><br />
+    For more information, see [Using `.env` files](/guides/development-environment/environment-variables#using-env-files).
+
+    </Admonition>
+4. Install `prisma` and `@prisma/client` version 3.15.2 or later.
+
+    ```terminal
+    npm install prisma@latest --save-dev 
+    npm install @prisma/client@latest --save 
+    ```
+
+5. Generate Prisma Client with the `--data-proxy` option.
+    
+    ```terminal
+    npx prisma generate --data-proxy
+    ```
+
+    The generated Prisma Client is a lightweight version because it excludes the local [query engine](/concepts/components/prisma-engines/query-engine) files. The Data Proxy handles the query engine logic.
+6. Instantiate Prisma Client in your application code.
```

Comment:
```suggestion
1. Install `prisma` and `@prisma/client` version 3.15.2 or later.

    ```terminal
    npm install prisma@latest --save-dev 
    npm install @prisma/client@latest --save 
    ```

1. Generate Prisma Client with the `--data-proxy` option.
    
    ```terminal
    npx prisma generate --data-proxy
    ```

    The generated Prisma Client is a lightweight version because it excludes the local [query engine](/concepts/components/prisma-engines/query-engine) files. The Data Proxy handles the query engine logic.
1. Instantiate Prisma Client in your application code.
```